src/test/java/com/example/universe/simulator/entityservice/unit/services/PlanetServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package com.example.universe.simulator.entityservice.unit.services;
+
+import com.example.universe.simulator.entityservice.entities.Planet;
+import com.example.universe.simulator.entityservice.exception.AppException;
+import com.example.universe.simulator.entityservice.exception.ErrorCodeType;
+import com.example.universe.simulator.entityservice.repositories.PlanetRepository;
+import com.example.universe.simulator.entityservice.services.PlanetService;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.never;
+
+@ExtendWith(MockitoExtension.class)
+class PlanetServiceTest {
+
+    @Mock
+    private PlanetRepository repo;
+
+    @InjectMocks
+    private PlanetService service;
+
+    private static UUID id;
+    private static Planet planet;
+    private static List<Planet> planets;
```

Comment:
Please remove these three instance variables and use local variables in each test method.